manifests/g/gileli121/windowtop/5.5.8/gileli121.windowtop.locale.en-US.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: gileli121.windowtop
+PackageVersion: 5.5.8
+PackageLocale: en-US
+Publisher: github.com/BiGilSoft
+PublisherUrl: https://github.com/BiGilSoft/WindowTop
+PublisherSupportUrl: https://github.com/BiGilSoft/WindowTop/issues
+# PrivacyUrl: 
```

Comment:
```suggestion
PrivacyUrl: https://windowtop.info/privacy-policy/
```

Reply:
Am i getting stalked? 👀

Reply:
> Am i getting stalked? 👀

Nopd :) I'm just bored and looking at the recent PR's to fill in additional fields